### PR TITLE
Added logic to invoke update dashboard flow from monitoring integration

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#385
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core @r0h4n @rishubhjain @GowthamShanmugam plz review

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -514,6 +551,22 @@ def sync_volumes(volumes, index, vol_options):
...
+            if not brick.exists():
+                job_id = monitoring_utils.update_dashboard(
+                    brick.brick_name,
+                    RESOURCE_TYPE_BRICK,
+                    NS.tendrl_context.integration_id,
+                    "add"
+                )
+                Event(
+                    Message(
+                        priority="info",
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: keep the priority as "debug" for all these logs, "info" is for user facing messages only and they put load on etcd. You know this issue already.

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 2]
```diff
@@ -346,6 +367,22 @@ def sync_volumes(volumes, index, vol_options):
...
             snapd_inited=volumes['volume%s.snapd_svc.inited' % index],
         )
+        if not volume.exists():
+            job_id = monitoring_utils.update_dashboard(
+                volume.name,
+                RESOURCE_TYPE_VOLUME,
+                NS.tendrl_context.integration_id,
+                "add"
+            )
+            Event(
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: use commons logger utils instead of event

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: i will change this

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: i will change this

**Pull request merged**